File: doc/Examples_SPL/expressionsCaesarJ/src/shortCircuit/Mult.java
cclass shortCircuit.ShortCircuit;

public cclass Mult {
	/**
	 * Method to evaluate a mult expression
	 */
	public int eval(){
		if (op1.eval() == 0){
            return 0;
        }else{
            return (this.op1.eval() * this.op2.eval());
        }//if
	}//eval

}//MultShortEval

File: trunk/Alejandro/Doc/Examples_SPL/PartialClassesExpressions/Expresiones/Eval/Mult.cs
using System;
using Expresiones;


namespace Expresiones
{

    public partial class Mult : IBinaryOperator
    {

        /**
         * Method to evaluate an expression
         * */
        public int eval()
        {
            return exp_izquierda.eval() * exp_derecha.eval();
        }//eval

    }//Mult
}//Expresiones